Bindings/Java/CMakeLists.txt

```diff
@@ -36,6 +35,13 @@ if( JAVA_COMPILE )
    )
 endif( JAVA_COMPILE )
 
-install(DIRECTORY Matlab DESTINATION Scripts 
-    PATTERN "Matlab/Dynamic_Walker_Example" EXCLUDE)
+install(DIRECTORY Matlab DESTINATION "${OPENSIM_INSTALL_SCRIPTSDIR}"
+    PATTERN "Matlab/Dynamic_Walker_Example" EXCLUDE
+    PATTERN "Matlab/configureOpenSim.m.in" EXCLUDE)
 
+# This script contains paths into the OpenSim installation that may be
+# different on different platforms, so we configure it with CMake variables.
+configure_file(Matlab/configureOpenSim.m.in
+    "${CMAKE_CURRENT_BINARY_DIR}/configureOpenSim.m" @ONLY)
+install(FILES "${CMAKE_CURRENT_BINARY_DIR}/configureOpenSim.m"
+        DESTINATION "${OPENSIM_INSTALL_SCRIPTSDIR}/Matlab")
```

Comment:
> Does the configuration leave the original .in file or does it overwrite it ? If it leaves the original, may be deleting it is a good idea ?

The configuration leaves the original `.in` file; it's part of the git repo, so it doesn't make sense to delete it. There's no overwriting. 



> Is it useful to create another variable (in the root CMakeLists.txt) holding the destination for Matlab (similarly for Python etc) stuff instead of hard-coding it here ?

Yes; I added a new variable for this.